runtime/flang/curdir.c

```diff
@@ -22,8 +22,6 @@
 #define getcwd _getcwd
```

Comment:
the function `__fort_getdir()` below for which this `getcwd` was defined doesn't seem to be used anywhere in flang and its runtime library. It's possible that there is more to delete in this file